### PR TITLE
Change warpsign permission

### PR DESCRIPTION
**Pull request by mathias-github**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,16 +60,25 @@ This plugin aims to port commands from Bukkit's Essentials to MCServer, but also
...
+Warpname
 
-###Enchant Sign template
+###Enchant Sign Template
 [Enchant]  
 EnchantmentID  
 EnchantmentLevel  
 RequiredXPLevelToEnchant
+
+###Portals
```
Comment (NiLSPACE) [inline README.md]: I think these should be [AdditionalInformation tables in the `Info.lua` file](http://mc-server.xoft.cz/LuaAPI/InfoFile.html#AdditionalInformation)?

**Comment by mathias-github** [comment]
I can add the info there as well, but I think the README is what people usually look at first.

**Comment by NiLSPACE** [comment]
The `Info.lua` file is used to generate both forum and Github readme files.

**Comment by mathias-github** [comment]
I've now added additional information and permissions to the Info.lua-file.

File: Info.lua [hunk 2]
```diff
@@ -6,8 +6,40 @@ g_PluginInfo =
...
+		{
+			Title = "Enchant Signs",
+			Contents = "To create an enchant sign, use the following template:
+			
+			[Enchant]
+			EnchantmentID
+			EnchantmentLevel
+			RequiredXPLevelToEnchant
+			
+			",
```
Comment (NiLSPACE) [inline Info.lua]: This won't work. If a string is goes over multiple lines you have to use
```lua
Contents = [[
The content goes here.
]]

Also, were the empty lines placed there for a reason?

**Comment by mathias-github** [comment]
AdditionalInfo should be fixed now. If it isn't, just tell me.

**Comment by NiLSPACE** [comment]
It seems our documentation about the `Info.lua` file is wrong. It isn't `AdditionalInformation`, but `AdditionalInfo`. This isn't your fault, but could you perhaps change that?

**Comment by mathias-github** [comment]
Changed

**Comment by mathias-github** [comment]
Seems like the files were changed, I'll create a new pull request

**Comment by mathias-github** [comment]
This can be closed now.

**Pull request closed without merging**